modules/apps/foundation/login/login-web/src/main/resources/META-INF/resources/create_account.jsp

```diff
@@ -130,77 +132,131 @@ birthdayCalendar.set(Calendar.YEAR, 1970);
-				<aui:input label="password" name="password1" size="30" type="password" value="" />
+				<c:if test="<%= PropsValues.CAPTCHA_CHECK_PORTAL_CREATE_ACCOUNT %>">
+					<portlet:resourceURL id="/login/captcha" var="captchaURL" />
 
-				<aui:input label="enter-again" name="password2" size="30" type="password" value="">
-					<aui:validator name="equalTo">
-						'#<portlet:namespace />password1'
-					</aui:validator>
-				</aui:input>
-			</c:if>
-
-			<c:choose>
-				<c:when test="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.FIELD_ENABLE_COM_LIFERAY_PORTAL_MODEL_CONTACT_BIRTHDAY) %>">
-					<aui:input name="birthday" value="<%= birthdayCalendar %>" />
-				</c:when>
-				<c:otherwise>
-					<aui:input name="birthdayMonth" type="hidden" value="<%= Calendar.JANUARY %>" />
-					<aui:input name="birthdayDay" type="hidden" value="1" />
-					<aui:input name="birthdayYear" type="hidden" value="1970" />
-				</c:otherwise>
-			</c:choose>
-
-			<c:if test="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.FIELD_ENABLE_COM_LIFERAY_PORTAL_MODEL_CONTACT_MALE) %>">
-				<aui:select label="gender" name="male">
-					<aui:option label="male" value="1" />
-					<aui:option label="female" selected="<%= !male %>" value="0" />
-				</aui:select>
-			</c:if>
-
-			<c:if test="<%= PropsValues.CAPTCHA_CHECK_PORTAL_CREATE_ACCOUNT %>">
-				<portlet:resourceURL id="/login/captcha" var="captchaURL" />
-
-				<liferay-ui:captcha url="<%= captchaURL %>" />
-			</c:if>
-		</aui:col>
-	</aui:fieldset>
-
-	<aui:button-row>
-		<aui:button cssClass="btn-lg" type="submit" />
-	</aui:button-row>
+					<liferay-ui:captcha url="<%= captchaURL %>" />
+				</c:if>
+			</aui:col>
+		</aui:fieldset>
+	</aui:fieldset-group>
 </aui:form>
 
-<liferay-util:include page="/navigation.jsp" servletContext="<%= application %>" />
\ No newline at end of file
+<aui:script>
+	var dialog = Liferay.Util.getWindow();
+
+	dialog.addToolbar(
+		[
+			{
+				cssClass: 'btn-lg btn-primary',
+				label: Liferay.Language.get('save'),
+				on: {
+					click: function() {
+						var form = AUI.$(document.<portlet:namespace />fm);
+
+						var formValidator = Liferay.Form.get('<portlet:namespace />fm').formValidator;
+
+						formValidator.validate();
+
+						if (!formValidator.hasErrors()) {
+							form.ajaxSubmit(
+								{
+									success: function(responseData) {
+										document.open();
+										document.write(responseData);
+										document.close();
+
+										var alert = $(responseData).find('.lfr-alert-container');
+
+										var alertSuccess = alert.find('.alert-success');
+
+										if (alertSuccess.length) {
+											dialog.fire(
+												'closeWindow',
+												{
+													alert: alert
+												}
+											);
+										}
+									}
+								}
+							);
+						}
+					}
+				}
+			},
+			{
+				cssClass: 'btn-lg btn-link close-modal',
+				label: Liferay.Language.get('cancel'),
```

Comment:
Also, please run the check-source-formatting module against your code. Having `Liferay.Language.get` calls in JSPs isn't allowed (it will lock up the browser), but the module will also give you a heads up about it.